server/backend/Services/YngdiengService.SearchV2.cs

```diff
@@ -34,9 +34,11 @@ public partial class YngdiengService : Yngdieng.Protos.YngdiengService.YngdiengS
                 resultCards.Add(GenericMessageCard("你正在試用榕典搜索V2。如遇問題，請將截圖和網址發送到 radium@mindong.asia。"));
```

Comment:
nit: can you split the changes to *.cs files into a different branch?  (These are just formatting changes and should not be part of this PR)

You can try this:

```
git checkout master
git checkout -b format-cs
git checkout nixl/audio-dialog -- server/
git commit -a -n -m "Format *.cs files"
gh pr create ....... # I recommend you to install cli.github.com

git checkout nixl/audio-dialog
git checkout master -- server/

```

Reply:
(this is not very important; feel free to ignore)